client/tests/acceptance/pageObjects/loginPage.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  url: function () {
+    return this.api.launchUrl + "/login";
+  },
+  commands: {
+    logIn: function (email, password) {
+      return this.waitForElementVisible("@emailInput")
+        .setValue("@emailInput", email)
+        .waitForElementVisible("@passwordInput")
+        .setValue("@passwordInput", password)
+        .waitForElementVisible("@loginBtn")
+        .click("@loginBtn");
+    },
+    isDashboardPage: async function () {
+      let result = false;
+      await this.waitForElementVisible("@dashboardHeader");
+      await this.isVisible("@dashboardHeader", (res) => {
+        result = res.value;
+      });
+      return result;
+    },
+  },
```

Comment:
Is this performed on the login page 🤔 ? `loginPage` page-object should only include interactions for the login page.